src/tool/subcommands/benchmark_cmd.rs

```diff
@@ -197,7 +197,10 @@ async fn benchmark_forest_encoding(
     let file = tokio::io::BufReader::new(File::open(&input).await?);
 
     let mut block_stream = CarStream::new(file).await?;
-    let roots = std::mem::take(&mut block_stream.header.roots);
+    let roots = std::mem::replace(
```

Comment:
Why not `impl<T: Default> Default for nunny:Vec`?

Reply:
I think a non-empty default isn't very natural